client/src/js/services/PrototypeApiService.js

```diff
@@ -161,4 +163,32 @@ function PrototypeApiService($http, util) {
     return this.$http.delete(target)
       .then(util.unwrapHttpResponse);
   }
+
+  /**
+   * @method reports
+   *
+   * @description
+   * Sends an HTTP GET request to the url "/route/reports/:id"
```

Comment:
The service concat the string 'reports/:param' to the service url, in our case 
`service.url = '/vouchers/'` 

The $http request is sended `to /vouchers/reports/:param`

![voucher route](https://cloud.githubusercontent.com/assets/5445251/16410812/145c8c14-3cd9-11e6-84f8-ac73256e0953.png)